### PR TITLE
Add contentType documentation

### PR DESCRIPTION
**Pull request by peyerluk**
ReleaseNotes PR that goes with this: https://github.com/upfrontIO/livingdocs-release-notes/pull/9

### CODE REVIEW

File: reference-docs/server-configuration/channel-config.md [hunk 1]
```diff
@@ -1,154 +1,71 @@
...
-    design: 'basic',
-    layout: 'default'
-  },
-  target: [
-    design: 'basic',
-    layout: 'default',
-    options: {
-      copyUnknownComponents: false
+  // Copy Configuration
+  copy: [
```
Comment (mthpvg) [inline reference-docs/server-configuration/channel-config.md]: It looks like `copy` is an array of objects like:
```js
copy: [{}, {}]
```
coffeescript trap 😄

File: reference-docs/server-configuration/content-type-config.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+```js
+const CheerioHtml = require('../../../lib/render-pipeline/output/cheerio_html')
+
+module.exports = {
+  // a label
+  'web': {
+    output: {
+      // an output renderer
+      'html': {
+        outputRenderer: new CheerioHtml({
```
Comment (mthpvg) [inline reference-docs/server-configuration/content-type-config.md]: According to https://github.com/upfrontIO/livingdocs-server/pull/1696/, we ditched `outputRenderer`.

File: reference-docs/server-configuration/content-type-config.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        }),
+        // resolve definitions for doc-includes
+        resolveIncludes: ['embed-teaser', 'list', 'categoryList']
+      }
+    }
+  },
+  'app': {
+    output: {
+      'json': {
+        outputRenderer: appRenderer
```
Comment (mthpvg) [inline reference-docs/server-configuration/content-type-config.md]: Same here.

File: walkthroughs/metadata/metadata-examples.md [hunk 4]
```diff
@@ -117,9 +117,9 @@ In `app/search/custom-mappings/metadata.json`, add the `slug` property:
 }
 ```
 
-Then, we extend the configuration of the **web** channel for documents of type **article**.
+Then, we extend the configuration of the **web** channel for documents with `conentType` **article**.
```
Comment (mthpvg) [inline walkthroughs/metadata/metadata-examples.md]: Typo on contentType.

File: reference-docs/server-configuration/content-type-config.md (hunk 2 above)
Comment (peyerluk) [reply reference-docs/server-configuration/content-type-config.md]: Both are possible. in fact if you wan to specify `resolveIncludes` you have to use the `outputRenderer` option.

Comment (mthpvg) [reply reference-docs/server-configuration/content-type-config.md]: Good to know 👍

**Pull request merged**